### DISCUSSION TITLE
Extracting inter-frame time from btrack .h5 file

### DISCUSSION THREAD

**Discussion by K-Meech** [quantumjot/btrack#411] [Q&A] [answered]
Hi all,

I'm processing some output files from btrack (`.h5`) and I was wondering if the inter-frame time is stored anywhere inside? (i.e. the time in seconds that each frame represents in the original image) I don't have access to the original images so I was curious if any of this kind of image metadata was captured directly in the btrack output.

Thanks for your help!

**Comment by quantumjot** [comment, marked as answer]
Hi @K-Meech! Unless the timestamps were explicitly added as metadata, then there's no guarantee that they will be stored. The file format uses an integer index by default for the frames.

**Comment by K-Meech** [comment]
Thanks for the info @quantumjot !
